src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -214,7 +256,9 @@ Status CreateRequestQueue::ProcessRequests() {
 
   // If we make it here, then there is nothing left in the queue. It's safe to
   // run new tasks again.
-  RAY_UNUSED(on_object_creation_blocked_callback_(ray::Priority()));
+  if(!RayConfig::instance().enable_BlockandEvictTasks() || RayConfig::instance().enable_BlockTasks()){
```

Comment:
Not sure if this is right. Shouldn't we call the callback when we are back under the threshold, inside the main loop?

Reply:
1. ProcessRequest() will set if we are under or over the threshold.
2. If we are over the threshold, we should call BlockTasks() on the lower priority object currently in the object store.
3. If we are under the threshold, then we should call BlockTasks(Priority()).

This means we also need to track what the lowest priority object currently is.